mutationCommands/CommandRegistry.js

```diff
@@ -60,17 +60,34 @@ function isArithmeticExpression(node) {
     }
 
     /**
+     * returns the command codes of all available mutation commands
+     * @returns {[string]} a list of mutation codes
+     */
+    function getAllCommandCodes() {
```

Comment:
Why not just do ```return _.keys(getDefaultExcludes());```?

I've written some unit tests and this seems to work just fine - keeps it DRY